### ISSUE TITLE
Tools: TestGPSGlitch.py: TabError: inconsistent use of tabs and spaces in indentation

### ISSUE THREAD

**Issue by cclauss** [ArduPilot/ardupilot#6997]
$ __python3 Tools/LogAnalyzer/tests/TestGPSGlitch.py__
```
  File "Tools/LogAnalyzer/tests/TestGPSGlitch.py", line 12
    def findSatsChan(self, channels):
                                    ^
TabError: inconsistent use of tabs and spaces in indentation
```

**Comment by OXINARF** [comment]
The best way is always to check what other files in the same folder do. I would prefer spaces and it appears the other files do use spaces, so I would say to go with spaces.

**Comment by cclauss** [comment]
Closed via #7017
